documentation/asciidoc/accessories/iqaudio/configuration.adoc

```diff
@@ -15,12 +17,14 @@ If you need to configure Raspberry Pi OS yourself, you can edit the boot configu
 | Codec Zero | `dtoverlay=iqaudio-codec`
 |===
 
+Alternatively, the `/boot/config.txt` file can be edited directly onto the Raspberry Pi Boot SD card inserted into your usual computer.
+
 === Disabling on-board audio
 
-To make your IQaudio sound card the primary audio device in Raspberry Pi OS you will need to
+To make your Raspberry Pi sound card the primary audio device in Raspberry Pi OS you will need to
 disable the Raspberry Pi’s on-board audio card. This is done simply by commenting out the
```

Comment:
"on-board audio card" doesn't make sense, so should probably be "on-board audio device" or just "on-board audio" or "built-in audio" or "internal audio device" ?